jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -307,8 +307,14 @@ class DeploymentFlow extends React.Component {
       credential: this.state.credential,
       region: this.state.region
     };
-    if (this.state.sshKey) {
-      args.config['authorized-keys'] = this.state.sshKey;
+    if (this.state.sshKeys) {
+      // Guarantee we have at least one key.
+      args.config['authorized-keys'] = this.state.sshKeys[0].text;
```

Comment:
I'm not really clear why this is necessary? Is there a difference between adding the keys after the model is created than when it's deployed?